gateway/managers/syncman/operations.go

```diff
@@ -72,6 +70,22 @@ func (s *Manager) setProject(ctx context.Context, project *config.Project) error
 	return s.store.SetProject(ctx, project)
 }
 
+// SetClusterConfig applies the set cluster config
+func (s *Manager) SetClusterConfig(ctx context.Context, req *config.ClusterConfig) error {
```

Comment:
This should return the status too. Each syncman operation now returns a status as well.